README.md

```diff
@@ -39,8 +39,8 @@ The following env variables need to be passed to the container:
 
 * `SMTP_SERVER` Server address of the SMTP server to use.
 * `SMTP_PORT` (Optional, Default value: 587) Port address of the SMTP server to use.
-* `SMTP_USERNAME` Username to authenticate with.
-* `SMTP_PASSWORD` Password of the SMTP user. If `SMTP_PASSWORD_FILE` is set, not needed.
+* `SMTP_USERNAME` (Optional) Username to authenticate with.
+* `SMTP_PASSWORD` (Optional) Password of the SMTP user. If `SMTP_PASSWORD_FILE` is set, not needed.
```

Comment:
You forgot to document it as mandatory if SMTP_USER is set like in the example env file.

Reply:
fixed!